tests/integration/test_integration.py

```diff
@@ -45,11 +45,11 @@
 }
 
 NUM_VAULT_UNITS = 3
+VAULT_UNIT_DOMAIN = "vault-k8s-endpoints.vault.svc.cluster.local"
```

Comment:
Where is this used?

Reply:
at some point it was used to build the requests to the vault units, but i started extracting those from the status. Removed.